docs/REFERENCE.md

```diff
@@ -1,51 +1,51 @@
+| [Notion](https://www.elastic.co/guide/en/elasticsearch/reference/current/es-connectors-notion.html) | **GA** | 8.14+ | 8.14+ | - | - | - | [View code](https://github.com/elastic/connectors/tree/main/connectors/sources/notion.py) |
+| [OneDrive](https://www.elastic.co/guide/en/elasticsearch/reference/current/es-connectors-onedrive.html) | **GA** | 8.11+ | 8.11+ | 8.11+ | 8.13+ | 8.11+ | [View code](https://github.com/elastic/connectors/tree/main/connectors/sources/onedrive.py) |
+| [Opentext Documentum](https://www.elastic.co/guide/en/elasticsearch/reference/current/es-connectors-opentext.html) | **Example** | n/a | n/a | n/a | n/a | - | [View code](https://github.com/elastic/connectors/tree/main/connectors/sources/opentext_documentum.py) |
+| [Oracle](https://www.elastic.co/guide/en/elasticsearch/reference/current/es-connectors-oracle.html) | **GA** | 8.12+ | - | - | - | - | [View code](https://github.com/elastic/connectors/tree/main/connectors/sources/oracle.py) |
+| [Outlook](https://www.elastic.co/guide/en/elasticsearch/reference/current/es-connectors-outlook.html) | **GA** | 8.13+ | - | 8.11+ | 8.13+ | 8.14+ | [View code](https://github.com/elastic/connectors/tree/main/connectors/sources/outlook.py) |
+| [PostgreSQL](https://www.elastic.co/guide/en/elasticsearch/reference/current/es-connectors-postgresql.html) | **GA** | 8.8+ | 8.11+ | - | - | - | [View code](https://github.com/elastic/connectors/tree/main/connectors/sources/postgresql.py) |
+| [Redis](https://www.elastic.co/guide/en/elasticsearch/reference/current/es-connectors-redis.html) | **Preview** | - | - | - | - | - | [View code](https://github.com/elastic/connectors/tree/main/connectors/sources/redis.py) |
+| [Amazon S3](https://www.elastic.co/guide/en/elasticsearch/reference/current/es-connectors-s3.html) | **GA** | 8.12+ | 8.12+ | 8.11+ | - | - | [View code](https://github.com/elastic/connectors/tree/main/connectors/sources/s3.py) |
+| [Salesforce](https://www.elastic.co/guide/en/elasticsearch/reference/current/es-connectors-salesforce.html) | **GA** | 8.12+ | 8.12+ | 8.11+ | 8.13+ | 8.13+ | [View code](https://github.com/elastic/connectors/tree/main/connectors/sources/salesforce.py) |
+| [ServiceNow](https://www.elastic.co/guide/en/elasticsearch/reference/current/es-connectors-servicenow.html) | **GA** | 8.10+ | 8.10+ | 8.11+ | 8.13+ | 8.13+ | [View code](https://github.com/elastic/connectors/tree/main/connectors/sources/servicenow.py) |
+| [Sharepoint Online](https://www.elastic.co/guide/en/elasticsearch/reference/current/es-connectors-sharepoint-online.html) | **GA** | 8.9+ | 8.9+ | 8.9+ | 8.9+ | 8.9+ | [View code](https://github.com/elastic/connectors/tree/main/connectors/sources/sharepoint_online.py) |
+| [Sharepoint Server](https://www.elastic.co/guide/en/elasticsearch/reference/current/es-connectors-sharepoint.html) | **Beta** | 8.15+ | - | 8.11+ | 8.13+ | 8.14+ | [View code](https://github.com/elastic/connectors/tree/main/connectors/sources/sharepoint_server.py) |
```

Comment:
```suggestion
| [Sharepoint Server](https://www.elastic.co/guide/en/elasticsearch/reference/current/es-connectors-sharepoint.html) | **Beta** | 8.15+ | - | 8.11+ | 8.13+ | 8.15+ | [View code](https://github.com/elastic/connectors/tree/main/connectors/sources/sharepoint_server.py) |
```

Reply:
🚗 related to https://github.com/elastic/elasticsearch/pull/116661/files#diff-36b646ff1a44f8213ed59dc5252846dedda7afb05d8e2fb5310754bd99077b5aR47